src/web/services/table-comparator.service.ts

```diff
@@ -11,6 +12,24 @@ export class TableComparatorService {
 
   constructor() { }
 
+  /**
+   * Compares two strings which are expected to be dates depending on order given.
+   * Assumes that the strings are formatted in the format 'ddd, DD MMM YYYY hh:mm:ss A'.
+   */
+  compareChronologically(strA: string, strB: string, order: SortOrder): number {
+    const dateFormat: string = 'ddd, DD MMM YYYY hh:mm:ss A';
```

Comment:
Should not be a function level constant? Perhaps you wanna set it at at least class level and reuse it elsewhere too... I see at least two occurrences of this string and if one is modified not the other, things might break.

Reply:
Changes made